### DISCUSSION TITLE
Install Azuracast with Cyberpanel

### DISCUSSION THREAD

**Discussion by vpsinghbaghel** [AzuraCast/AzuraCast#4307] [Q&A]
Can we run docker version of Azuracast on CyberPanel server through Docker Manager option available in CyberPanel ?

Server Type: VPS
Web Control Panel: CyberPanel
OS: CentOS 7/8 or Ubuntu 18/20.

**Comment by ItsMitchh** [comment]
Hello,

As long as it's not running on ports: (80, 443) It should be able to run fine, we have some documentation to assist you further. 

https://docs.azuracast.com/en/user-guide/troubleshooting

You can always change the ports in the custom .env files we provide.

**Reply by Oric48k** [reply]
Hi all, 

So when I tried to install Azuracast_web_v2 as a container on Nextcloud it wouldn't work. 
I'm not mapping a new path when creating the container but would like to eventually.

Anyone seen this before?

Starting up AzuraCast services...

Warning: Redis::connect(): php_network_getaddresses: getaddrinfo for redis failed: Name or service not known in /var/azuracast/www/bin/uptime_wait on line 134

Call Stack:
    0.0004     416880   1. {main}() /var/azuracast/www/bin/uptime_wait:0
    0.0005     419672   2. UptimeWait->run() /var/azuracast/www/bin/uptime_wait:167
    0.0005     419704   3. UptimeWait->checkRedis() /var/azuracast/www/bin/uptime_wait:77
    0.0005     419752   4. Redis->connect($host = 'redis', $port = 6379, $timeout = 15) /var/azuracast/www/bin/uptime_wait:134

PHP Warning:  Redis::connect(): php_network_getaddresses: getaddrinfo for redis failed: Name or service not known in /var/azuracast/www/bin/uptime_wait on line 134
PHP Stack trace:
PHP   1. {main}() /var/azuracast/www/bin/uptime_wait:0
PHP   2. UptimeWait->run() /var/azuracast/www/bin/uptime_wait:167
PHP   3. UptimeWait->checkRedis() /var/azuracast/www/bin/uptime_wait:77
PHP   4. Redis->connect($host = 'redis', $port = 6379, $timeout = 15) /var/azuracast/www/bin/uptime_wait:134
PHP Warning:  Redis::connect(): php_network_getaddresses: getaddrinfo for redis failed: Name or service not known in /var/azuracast/www/bin/uptime_wait on line 134
PHP Stack trace:
PHP   1. {main}() /var/azuracast/www/bin/uptime_wait:0
PHP   2. UptimeWait->run() /var/azuracast/www/bin/uptime_wait:167
PHP   3. UptimeWait->checkRedis() /var/azuracast/www/bin/uptime_wait:77
PHP   4. Redis->connect($host = 'redis', $port = 6379, $timeout = 15) /var/azuracast/www/bin/uptime_wait:134
[0G🖥️🎶－🎵－📻 Waiting...
Warning: Redis::connect(): php_network_getaddresses: getaddrinfo for redis failed: Name or service not known in /var/azuracast/www/bin/uptime_wait on line 134

Call Stack:
    0.0004     416880   1. {main}() /var/azuracast/www/bin/uptime_wait:0
    0.0005     419672   2. UptimeWait->run() /var/azuracast/www/bin/uptime_wait:167
    1.0121     419960   3. UptimeWait->checkRedis() /var/azuracast/www/bin/uptime_wait:77
    1.0121     420008   4. Redis->connect($host = 'redis', $port = 6379, $timeout = 15) /var/azuracast/www/bin/uptime_wait:134

[0G🖥️－🎶－🎵📻 Waiting...
Warning: Redis::connect(): php_network_getaddresses: getaddrinfo for redis failed: Name or service not known in /var/azuracast/www/bin/uptime_wait on line 134

Call Stack:
    0.0004     416880   1. {main}() /var/azuracast/www/bin/uptime_wait:0
    0.0005     419672   2. UptimeWait->run() /var/azuracast/www/bin/uptime_wait:167
    2.0228     419960   3. UptimeWait->checkRedis() /var/azuracast/www/bin/uptime_wait:77
    2.0228     420008   4. Redis->connect($host = 'redis', $port = 6379, $timeout = 15) /var/azuracast/www/bin/uptime_wait:134

PHP Warning:  Redis::connect(): php_network_getaddresses: getaddrinfo for redis failed: Name or service not known in /var/azuracast/www/bin/uptime_wait on line 134
PHP Stack trace:
PHP   1. {main}() /var/azuracast/www/bin/uptime_wait:0
PHP   2. UptimeWait->run() /var/azuracast/www/bin/uptime_wait:167
PHP   3. UptimeWait->checkRedis() /var/azuracast/www/bin/uptime_wait:77
PHP   4. Redis->connect($host = 'redis', $port = 6379, $timeout = 15) /var/azuracast/www/bin/uptime_wait:134
[0G🖥️🎵－🎶－📻 Waiting...
Warning: Redis::connect(): php_network_getaddresses: getaddrinfo for redis failed: Name or service not known in /var/azuracast/www/bin/uptime_wait on line 134
